src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -46,9 +46,12 @@ internal static RootCommand CreateRootCommand()
             var versionOption = new Option<string?>("--version", "OpenAPI specification version");
             versionOption.AddAlias("-v");
 
+            var metadataVersionOption = new Option<string?>("--metadata-version", "Graph metadata version to use. Defaults to v1.0");
```

Comment:
```suggestion
            var metadataVersionOption = new Option<string?>("--metadata-version", "Graph metadata version to use.");
```
I don't think we're defaulting this anywhere.